handler/PostMessageSentHandler.ts

```diff
@@ -86,8 +86,9 @@ export class PostMessageSentHandler {
         const { visitor: { token: visitorToken } } = room as ILivechatRoom;
 
         try {
-            await botTypingListener(rid, this.modify.getNotifier().typing({ id: rid, username: DialogflowBotUsername }));
-            response = (await Dialogflow.sendRequest(this.http, this.read, this.modify, this.persistence, rid, text, DialogflowRequestType.MESSAGE));
+            await botTypingListener(this.modify, rid, DialogflowBotUsername);
+            response = (await Dialogflow.sendRequest(this.http, this.read, this.modify, rid, text, DialogflowRequestType.MESSAGE));
```

Comment:
4th argument `this.persistence` is missing here. Isn't it giving you an error?

Reply:
@Shailesh351 you can ignore this PR, I will make the changes in a new one. Should I pull from cx_development or master?

Reply:
One PR for merging master in cx_development. And one for this mediaCardURL?

Reply:
I want to do a small PR to fix Steve's issue from yesterday. I will just update this PR to be only about changes for mediaCardURL.